src/extension/actions.js

```diff
@@ -80,6 +63,39 @@ async function updateAuthToken({
   return 'invalid message';
 }
 
+/**
+ * Shows the sidekick if it is hidden.
+ */
+export async function showSidekickIfHidden() {
+  const visible = await getDisplay();
+  if (!visible) {
+    await setDisplay(true);
+    // wait for sidekick to be visible and check tab to be called by storage listener
+    // eslint-disable-next-line no-promise-executor-return
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
I don't love this and am open to other options. The problem is calling `setDisplay(true)` is not enough to get the sidekick to appear. `checkTab` must also be called. I am not able to import `checkTab` into this file due to the circular dependancy it will introduce. So instead I'm relying on the [listener we already have setup on storage](https://github.com/adobe/aem-sidekick/pull/295/files#diff-15c7d9d58f901c53a5b2c441ddb22b96e61026245ea8b50df839cd0b5c34af36R58) to call `checkTab`.